modules/fts/pages/fts-rebalance-failover.adoc

```diff
@@ -5,25 +5,30 @@ The search service maintains a cluster-wide set of index definitions and metadat
 
 == Handling partitions during a rebalance
 
-The search service redistributes the index partitions across the available search service nodes for a balanced partition-node assignment during a rebalance operation. The newly assigned index partitions are built afresh over DCP feed on the new nodes. And once the new partitions are built up to the current / latest sequence numbers, they are promoted to take the live traffic, and the older partitions are deleted from the system. 
+The search service redistributes the index partitions across the available search service nodes for a balanced partition-node assignment during a rebalance operation. The newly assigned index partitions are built afresh over DCP feed on the new nodes. Once the new partitions are built up to the current / latest sequence numbers, the partitions are promoted to handle live traffic, and the older partitions are deleted from the system. 
```

Comment:
afresh over DCP -> "utilizing the DCP"  feed 
"built up to the current / latest sequence numbers" -> "built up to the most current or latest sequence numbers"

Reply:
sounds good!

Reply:
Updated